Make BEIS and DoF logos visible for NIR respondents.

samiwel opened the pull request:
### What is the context of this PR?
BEIS and the NI department of finance have requested that the NI department of finance and BEIS logos are present on the NI UKIS questionnaire landing page.

This PR ensures that both logos are visible when UKIS survey is launched with a Northern Ireland Region code.

Here is what Northern Ireland respondents should see:
![image](https://cloud.githubusercontent.com/assets/4097796/22330133/8c68d5fc-e3bc-11e6-9da9-95acb271b238.png)


Here is what respondents from other regions should see:
![image](https://cloud.githubusercontent.com/assets/4097796/22330100/6651e534-e3bc-11e6-9eff-bc34044a3276.png)


### How to review 
The UKIS landing page has both the NI and BEIS logo present when launched in the Northern Ireland region.
The NI logo is not present in non-Northern Ireland regions (i.e. just the BEIS logo is shown)

samiwel committed:
Make BEIS and DoF logos visible for NIR respondents.

collisdigital commented:
Can you attach a screenshot. Note the BEIS logo has changed (see Trello).

samiwel commented:
Sure, I'll attach a screenshot once I've replaced the logo.

samiwel committed:
Add new logo and adjust size.

collisdigital commented on app/themes/ukis/templates/introduction.html:
```diff
@@ -12,9 +12,10 @@
 {% block business_details %}
   <div class="u-mb-m">
     {% if meta.survey.region_code == 'GB-NIR' %}
+      <img src="{{ theme_static('logo-beis.png') }}" alt="Department for Business, Energy and Industrial Strategy" width="200" />
```
If you swap the order of the IF statement you could always have the BEIS logo and only add the NI logo inside the IF; no need for an ELSE.

samiwel committed:
Refactor template logic.

samiwel committed:
Merge branch 'master' into eq-883-ni-and-beis-logo-visible-for-nir

ajmaddaford commented:
The `gy` on the end of 'Strategy' in the BEIS logo is slightly cut off for me
![image](https://cloud.githubusercontent.com/assets/1754822/22339811/2f4b407c-e3e3-11e6-8809-1c6f86fc6f86.png)

hamishtaplin reviewed (changes requested):
Can we get this closer to the design? There should be a margin between the logos and attention paid to how they stack on mobile. Also, the proportions of the logos are off—the BEIS one is too big and the file needs to be scaled down so we're not shipping a 900px wide image only to display it at 200px. Ideally it would be re-sized to 400px wide so it scales nicely for @2x display.

samiwel committed:
Make logos more closely match the design.

samiwel committed:
Merge branch 'master' into eq-883-ni-and-beis-logo-visible-for-nir

samiwel committed:
Remove inline css styles.

samiwel committed:
Merge branch 'master' into eq-883-ni-and-beis-logo-visible-for-nir

samiwel commented:
![image](https://cloud.githubusercontent.com/assets/4097796/22371180/94b79294-e48d-11e6-868f-47fd04dd72cf.png)

I've updated the styling so that it more closely resembles the design.
I've also reduced the size of the image to 400px and removed the inline CSS.

On mobile it stacks like this:

![image](https://cloud.githubusercontent.com/assets/4097796/22371236/eb45cc7a-e48d-11e6-8919-791f8d9d33ca.png)

samiwel committed:
Revert change to config.
